Detectors/TPC/calibration/macro/dumpDigits.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+usage() {
+  echo "Usage:"
+  echo "dumpDigits.sh <required arguments> [optional arguments]"
+  echo
+  echo
+  echo "required arguments"
+  echo "     -i, --fileInfo=     :  wildcard of input files in single quotes."
+  echo "                            Can obtain the number of time bins separated by a ':'"
+  echo "                            e.g. 'cru*.raw:1000' for 1000 time bins"
+  echo "                            the time bin option after the ':' overwrites the --timeBins option"
+  echo "    -p, --pedestalFile=  : pedestal and noise file"
+  echo
+  echo "optional arguments:"
+  echo "    -o, --outputFile=    : set output file name (default: '$outputFile')"
+  echo "    -f, --fistTimeBin=   : first time bin for pulser search"
+  echo "    -l, --lastTimeBin=   : last time bin for pulser search"
+  echo "    -n, --nevents=       : number of events to process (default: $nevents)"
+  echo "    -m, --adcMin=        : minimal ADC value accepted (default: $adcMin)"
+  echo "    -x, --adcMax=        : maximal ADC value accepted (default: $adcMax)"
+  echo "    -t, --threshold=     : threshold in nSigma noise (default: $threshold)"
+  echo "    -v, --verbosity=     : set verbosity level for raw reader"
+  echo "    -d, --debugLevel=    : set debug level for raw reader"
+  echo "    -h, --help           : show this help message"
+}
```

Comment:
you can use
```
cat<<EOF
Usage:
dumpDigits.sh <required arguments> [optional arguments]
...
EOF
```
which will allow you to drop `echo` and quotes

Reply:
Thanks, I'll do this.

Reply:
done